environments/site/inventory/groups

```diff
@@ -164,6 +164,9 @@ fatimage
 [extra_packages]
 # Hosts to install specified additional packages on
 
+[extra_packages:children]
```

Comment:
There's already an empty block directly above this for extra_packages, you just need to modify that.